panel/template/base.py

```diff
@@ -15,6 +15,7 @@
     Type,
 )
 
+import jinja2
```

Comment:
Maybe move this into the method. I think we need to run this file when `pip install -e .` because of `panel bundle`

Reply:
jinja2 is a core dependency of bokeh so we already satisfy the dependency in the build env.